maintainers/maintainer-list.nix

```diff
@@ -6056,6 +6056,12 @@
     githubId = 810075;
     name = "Juan Rodal";
   };
+  juboba = {
```

Comment:
This should be in a seperate commit.

Reply:
thank you @IvarWithoutBones , change made.